site/src/app/v2/trouver-un-club/components/club-finder/ClubFinder.tsx

```diff
@@ -183,12 +183,25 @@ const ClubFinder = ({ activities, isProVersion }: Props) => {
     setFocusOn(`#club-list > li:nth-child(${clubsOnList.firstRecievedClubIndex}) a`);
   }, [clubsOnList.firstRecievedClubIndex]);
 
+  useEffect(() => {
+    if ((cityParam && !postalCodeParam) || (!cityParam && postalCodeParam)) {
```

Comment:
```suggestion
    if (cityParam ^ postalCodeParam) {
```
Si j'ai bien compris, tu veux rentrer dedans que lorsqu'une des deux conditions est vraies, donc c'est un OU exclusif?